src/Display.tsx

```diff
@@ -14,6 +14,19 @@ interface VictimDetailProps {
 }
 
 const VictimDetail: React.FC<VictimDetailProps> = props => {
+  const [address, setAddress] = useState("");
+  fetch(
+    `https://api.mapbox.com/geocoding/v5/mapbox.places/${props.data.lat},${props.data.lon}.json?` +
```

Comment:
can we have `host` as constant

Reply:
> can we have `host` as constant

Should I store API url in const var instead of directly using string?

Reply:
Also, would it be better to use [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) instead?